components/data-table/private/row.jsx

```diff
@@ -64,11 +69,17 @@ class DataTableRow extends React.Component {
 
 	// ### Render
 	render() {
+		const ariaProps = {};
 		const isSelected = this.isSelected();
 
+		if (this.props.canSelectRows) {
+			ariaProps['aria-selected'] = isSelected ? 'true' : 'false';
```

Comment:
As part of meeting the new guidelines discussed in https://user-images.githubusercontent.com/1290832/48597728-6f45c680-e91c-11e8-86af-096d6e218213.png, I've made aria-selected present on all selectable rows, saying either 'true' or 'false' depending on whether they are selected. Is this appropriate @interactivellama ?

Reply:
Yes, that looks right.